app/controllers/application_controller/explorer.rb

```diff
@@ -221,7 +221,7 @@ def x_build_dynatree(options)
     # :full_ids               # stack parent id on top of each node id
 
     roots = x_get_tree_objects(options.merge(
-                                 :userid => session[:userid], # Userid for RBAC filtering
+                                 :user   => current_user,     # Userid for RBAC filtering
```

Comment:
@kbrock you should also consider adding user to https://github.com/ManageIQ/manageiq/blob/master/app/presenters/tree_builder.rb#L250 this is where all of the explorer trees are being built. app/controllers/application_controller/explorer.rb
 need to be cleaned up to remove all the obsolete code as we have moved most of the explorers tree building code into presenters.